eva-accession-pipeline/src/test/java/uk/ac/ebi/eva/accession/pipeline/io/AccessionWriterTest.java

```diff
@@ -188,4 +193,28 @@ public void createAccessionAndItAppearsInTheReportVcf() throws Exception {
         assertEquals(vcfLine.split("\t")[ACCESSION_COLUMN],
                      ACCESSION_PREFIX + accessions.iterator().next().getAccession());
     }
+
+    @Test
+    public void shouldThrowIfSomeVariantsWereNotAccessioned() {
+        SubmittedVariant variant = new SubmittedVariant("assembly", TAXONOMY, "project", "contig", START_1,
+                                                        REFERENCE_ALLELE, ALTERNATE_ALLELE, false);
+
+        thrown.expect(IllegalStateException.class);
+        accessionWriter.assertCountsMatch(Collections.singletonList(variant), new ArrayList<>());
+    }
+
+    @Test
+    public void shouldThrowIfSomeVariantsWereNotAccessionedInAChunkWithRepeatedVariants() {
```

Comment:
can you add a test similar to this, with repeated variants, but where all of them have been accessioned and no exception is thrown? If that case is already covered in previous tests, then is not needed.

Reply:
`saveSameAccessionTwice` checks a repeated variant is accessioned correctly, calling `AccessionWriter::write`, and this calls `AccessionWriter::assertCountsMatch`. So it's tested at class level, not method level. should I add another test?

Reply:
As is already tested I don't think is needed then.